### ISSUE TITLE
Blockchain wallet schema

### ISSUE THREAD

**Issue by T-Dnzt** [omisego/ewallet#690]
Depends on #689 

In this first implementation, the private key for the wallet can be encrypted and stored in the DB with the encryption key being stored in an environment variable. The security around the storage of the encryption key will be added in another issue.

The schema will at least need the following fields:

- `uuid`
- `id`
- `type`: `hot` / `cold`
- `encrypted_private_key`
- `public_key`
- `address`
- `primary`
- `timestamps`

A set of rules should also be available for this. Like balance shouldn't go above `xxx`, with a few options like notifying or transferring to a cold wallet.

**Comment by unnawut** [comment]
> the private key for the wallet can be stored in the DB

to

> the public key for the wallet can be stored in the DB

**Comment by unnawut** [comment]
This is a new table separated from the existing `wallet` table?

**Comment by T-Dnzt** [comment]
Yes, it is.

**Comment by unnawut** [comment]
Also... apart from the `hot` / `cold` value, any other different behaviors between the two?

**Comment by unnawut** [comment]
@T-Dnzt What's the purpose of `primary` field?

**Comment by unnawut** [comment]
> The `primary: true` wallet identifies the unique hot wallet used by default to receive funds from the blockchain.

**Comment by unnawut** [comment]
For reference, #806 PR was closed because it should be much simpler to add. Just need to add blockchain addresses and transaction ids to existing tables.
